core/encode.go

```diff
@@ -102,6 +102,22 @@ func EncodeAttestationParSignedData(att *eth2p0.Attestation, shareIdx int) (ParS
 	}, nil
 }
 
+// EncodeVoluntaryExitParSignedData encodes to json to pass between Go components losing typing,
+// returns a ParSignedData that contains json.
+// WARNING: using this method makes you lose Golang type safety features.
```

Comment:
please remove unprofessional comments

Reply:
What is unprofessional?

Reply:
Putting personal style disagreements into the code as comments

Reply:
This is not a personal style, this is a statement of fact, that other contributors should be aware of.

If you think this decision is shameful, we should decide about why we are doing it this way, when there are easier ways.

Reply:
You can find in Open Source software this type of comments very often, that using a method is dangerous for example.

Reply:
https://github.com/ethereum/go-ethereum/blob/07508ac0e9695df347b9dd00d418c25151fbb213/node/config.go#L176

Reply:
https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/accounts/scwallet/README.md?plain=1#L11

Reply:
https://github.com/ethereum/go-ethereum/blob/8a134014b4b370b4a3632e32a2fc8e84ee2b6947/crypto/bls12381/arithmetic_fallback.go#L26-L30

Reply:
https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/metrics/meter.go#L104

Reply:
https://github.com/ethereum/go-ethereum/blob/41671d449f5b3a19119eef070151ec72fdbb31f8/build/nsis.envvarupdate.nsh#L5-L6

Reply:
https://github.com/ethereum/go-ethereum/blob/1941c5e6c99689be2e27b3e537bd591688b717de/core/rawdb/chain_freezer.go#L46-L48